workflows/toast_sim_ground.py

```diff
@@ -325,15 +338,21 @@ def main():
             name="det_pointing_radec", quats="quats_radec"
         ),
         toast.ops.ScanHealpix(name="scan_map", enabled=False),
-        toast.ops.SimNoise(name="sim_noise"),
         toast.ops.SimAtmosphere(name="sim_atmosphere"),
+        toast.ops.TimeConstant(
```

Comment:
Should we add this to the example satellite workflow as well?

Reply:
Excellent point. Done!